### PR TITLE
Fixes for wired M-Bus

### PR DESCRIPTION
**Pull request by alexwoods**
- ensure port is not held open
- fix conversion function to return data!
- small logging additions

### CODE REVIEW

File: net.solarnetwork.node.io.mbus.jmbus/src/net/solarnetwork/node/io/mbus/jmbus/JMBusMBusNetwork.java [hunk 1]
```diff
@@ -69,7 +74,9 @@ public MBusData read(int address) throws IOException {
 			return null;
 		}
 		conn.open();
-		return conn.read();
+		MBusData result = conn.read();
+		conn.close();
+		return result;
```
Comment (msqr) [inline net.solarnetwork.node.io.mbus.jmbus/src/net/solarnetwork/node/io/mbus/jmbus/JMBusMBusNetwork.java]: Maybe this is clearer with a try-with-resource block like 
```java
try ( MBusConnection conn = createConnection(address) ) {
    conn.open();
    return conn.read();
}
```

**Review by msqr — CHANGES_REQUESTED** [review]
Looking good to me: just one small request to change to a try block.

File: net.solarnetwork.node.io.mbus.jmbus/src/net/solarnetwork/node/io/mbus/jmbus/JMBusMBusNetwork.java (hunk 1 above)
Comment (alexwoods) [reply net.solarnetwork.node.io.mbus.jmbus/src/net/solarnetwork/node/io/mbus/jmbus/JMBusMBusNetwork.java]: Oh yeah, that's much better. Thanks for bring that language feature to my attention :)
I've checked that change in.

**Review by msqr — APPROVED** [review]
Looks great, thanks for the fixes.

**Pull request merged**